libia2/ia2.c

```diff
@@ -64,7 +64,7 @@ int protect_tls_pages(struct dl_phdr_info *info, size_t size, void *data) {
     exit(-1);
   }
 
-  struct PhdrSearchArgs *search_args = (struct PhdrSearchArgs *)data;
+  struct PhdrSearchArgsTls *search_args = (struct PhdrSearchArgsTls *)data;
```

Comment:
Is this change even relevant any longer? I don't see any of this commit in the final diff.

Reply:
Will rebase away.